src/js/view/view.js

```diff
@@ -150,6 +150,7 @@ define([
                     width: containerWidth,
                     height: containerHeight
                 });
+                _this.trigger(events.JWPLAYER_BREAKPOINT,{ breakpoint: getBreakpoint(containerWidth) });
```

Comment:
Odd that the linter doesn't complain about this - no space after comma, no wrapping object literal.